backend/src/services/enhanced_google_places_service.py

```
@@ -157,14 +167,21 @@ async def search_restaurants(
             lat_lng = geocode_result[0]['geometry']['location']
             coordinates = (lat_lng['lat'], lat_lng['lng'])
 
-            # Use nearby search with coordinates for better location accuracy
+            # Use Text Search API (Places API New) instead of legacy nearby search
             search_radius = radius if radius else 5000  # Default 5km radius
```

Ruff (0.13.1)

Local variable `search_radius` is assigned to but never used

Remove assignment to unused variable `search_radius`

(F841)